<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winGameWizard</class>
 <widget class="QDialog" name="winGameWizard">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>404</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Game Set Up Wizard</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>591</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Introduction">
    <widget class="QDialogButtonBox" name="btnBxIntro">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>335</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>-117</y>
       <width>501</width>
       <height>361</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;OneLauncher Setup Wizard:&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;This wizard will quickly take you through the steps&lt;/p&gt;&lt;p&gt;needed to get up and running with OneLauncher. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="GameFinder">
    <widget class="QComboBox" name="cboGame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>350</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tblGame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>581</width>
       <height>231</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFind">
     <property name="geometry">
      <rect>
       <x>395</x>
       <y>22</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Find Games</string>
     </property>
    </widget>
    <widget class="QDialogButtonBox" name="btnBoxOptions">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>335</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblGame">
     <property name="geometry">
      <rect>
       <x>-125</x>
       <y>36</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Game</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblApplication">
     <property name="geometry">
      <rect>
       <x>-125</x>
       <y>0</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Application</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboApplication">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>4</y>
       <width>350</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnBoxOptions</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>winGameWizard</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBoxOptions</sender>
   <signal>rejected()</signal>
   <receiver>winGameWizard</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
